os/arch/Kconfig

```diff
@@ -587,7 +587,7 @@ config ARCH_INTERRUPTSTACK
 
 config ARCH_NESTED_IRQ_STACK_SIZE
 	int "Nested IRQ secondary stack size"
-	depends on ARCH_HAVE_INTERRUPTSTACK
+	depends on ARCH_HAVE_INTERRUPTSTACK && ARCH_ARMV8M_FAMILY
```

Comment:
Hi @kishore-sn, if menuconfig is used for other boards, we still get CONFIG_ARCH_NESTED_IRQ_STACK_SIZE in the config. How about adding dependency of the config on armv8 in the kconfig?

Reply:
done

Reply:
As far as I know, armv7-m and armv8-m are almost the same. Is it ok to apply only for v8?